### ISSUE TITLE
Some error occured when I use your pretrained model of 3DSSD

### ISSUE THREAD

**Issue by Xiangzhaohong** [tomztyang/3DSSD#6]
Hi @tomztyang 

root@53da94df9c78:~/3DSSD# python lib/core/evaluator.py --cfg configs/kitti/3dssd/3dssd.yaml --restore_model_path /root/data/3dssd
/usr/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: compiletime version 3.5 of module 'tensorflow.python.framework.fast_tensor_util' does not match runtime version 3.6
  return f(*args, **kwds)
/root/3DSSD/lib/core/config.py:514: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  yaml_cfg = AttrDict(yaml.load(f))
**** Saving Evaluation results to the path /root/data/3dssd ****
**** Dataset length is 3384 ****
numpy applys seed 2387376643
Start data provider ipc://@dataflow-map-pipe-b2d27482-b'0'
Starting Evaluation at 2020-04-29-06:14:36
No model found in /root/data/3dssd. Will try again in 300 seconds

In /root/data/3dssd/ is the pretrained models.
root@53da94df9c78:~/3DSSD# ls -l /root/data/3dssd/
total 36064
-rw-r--r-- 1 1000 1000      384 Apr 29 06:14 log_train.txt
drwxr-xr-x 2 root root     4096 Apr 29 06:08 model-75051
-rw-r--r-- 1 1000 1000 30222888 Apr  8 07:22 model-75051.data-00000-of-00001
-rw-r--r-- 1 1000 1000    20145 Apr  8 07:22 model-75051.index
-rw-r--r-- 1 1000 1000  6674647 Apr  8 07:22 model-75051.meta

Should I train my own model for testing?

**Comment by tomztyang** [comment]
Hi,
You have to assign the path to the checkpoint file. For example, 
--restore_model_path=/path/to/3dssd/model-xxx.ckpt.

**Comment by Xiangzhaohong** [comment]
Ok, I will train and test it .
Thanks for reply.

**Comment by deeptibhegde** [comment]
Hello, the pretrained model file does not have the ckpt file. Can you please provide this @tomztyang ?

**Comment by York1996OutLook** [comment]
Hello, the pretrained model file does not have the ckpt file. Can you please provide this @tomztyang ?
